### PR TITLE
Remove usage of compile-time Names constant

### PR DESCRIPTION
**Pull request by nadundesilva**

### CODE REVIEW

File: observelib/observe/src/main/java/org/ballerinalang/observe/nativeimpl/ObserveNativeImplConstants.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
     private ObserveNativeImplConstants() {
     }
 
-    public static final Module OBSERVE_PACKAGE_ID = new Module(BALLERINA_ORG.getValue(), "observe", "0.9.0");
+    public static final Module OBSERVE_PACKAGE_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, "observe", "0.9.0");
```
Comment (warunalakshitha) [inline observelib/observe/src/main/java/org/ballerinalang/observe/nativeimpl/ObserveNativeImplConstants.java:36]: May be we can get rid of this contant and use io.ballerina.runtime.Environment.getCurrentModule.

Comment (warunalakshitha) [reply observelib/observe/src/main/java/org/ballerinalang/observe/nativeimpl/ObserveNativeImplConstants.java:36]: https://github.com/ballerina-platform/module-ballerina-io/pull/49/files

**Comment by warunalakshitha** [comment]
LGTM

**Review by warunalakshitha — APPROVED** [review]

File: observelib/observe/src/main/java/org/ballerinalang/observe/nativeimpl/ObserveNativeImplConstants.java (hunk 1 above)
Comment (nadundesilva) [reply observelib/observe/src/main/java/org/ballerinalang/observe/nativeimpl/ObserveNativeImplConstants.java]: Fixed in e692962

**Pull request merged**